### ISSUE TITLE
Error installing 7-zip

### ISSUE THREAD

**Issue by sebistark** [Ac6Embedded/vscode-zephyr-workbench#2]
Hi, I had the following error during the Host Tool installing sequence on windows.
I downloaded and installed 7-zip manually and it worked.

```
----------------------------------------
                 7-Zip
----------------------------------------
7-Zip is not installed. Installing now...
Cannot index into a null array.
At C:\Users\xxx\.vscode\extensions\ac6.zephyr-workbench-0.1.1\scripts\hosttools\install.ps1:293 char:9
+         Download-FileWithHashCheck $SevenZ_array[0] $SevenZ_array[1]  ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : NullArray

Start-Process : This command cannot be run due to the error: The system cannot find the file specified.
At C:\Users\xxx\.vscode\extensions\ac6.zephyr-workbench-0.1.1\scripts\hosttools\install.ps1:297 char:9
+         Start-Process -FilePath $SevenZInstallerPath -ArgumentList "/ ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Start-Process], InvalidOperationException
    + FullyQualifiedErrorId : InvalidOperationException,Microsoft.PowerShell.Commands.StartProcessCommand

7-Zip installation completed.
ERROR: 7-Zip was not installed ! Stop here !!
4

 *  The terminal process "powershell.exe -Command powershell -ExecutionPolicy Bypass -File c:\Users\xxx\.vscode\extensions\ac6.zephyr-workbench-0.1.1\scripts\hosttools\install.ps1  -InstallDir C:\Users\sfavre" terminated with exit code: 1. 
 *  Terminal will be reused by tasks, press any key to close it. 
```

**Comment by RoyAc6** [comment]
Hi @sebistark, thank your for reporting this issue, it was fixed in the lastest release (0.1.2)

**Comment by RoyAc6** [comment]
Hi @sebistark, thank your for reporting this issue, it was fixed in the lastest release (0.1.2)
